byzcoin/api.go

```diff
@@ -89,6 +89,17 @@ func newLedgerWithClient(msg *CreateGenesisBlock, c *Client) (*CreateGenesisBloc
 	return reply, nil
 }
 
+// GetAllChainIDs returns the list of Byzcoin chains known by the server given in
+// parameter.
+func (c *Client) GetAllChainIDs(si *network.ServerIdentity) (*GetAllChainIDsResponse, error) {
```

Comment:
The name might cause a confusing because it's quite similar to `GetAllSkipChainIDs`, "chain" in our system is still skipchain. Why not call it `GetAllByzCoinChainIDs`?

Reply:
`GetAllByzCoinIDs`?